README.md

```diff
@@ -1,31 +1,39 @@
 CellMincer
 ===========
 
-CellMincer is a software package for learning self-supervised denoising models for voltage-imaging movies.
+CellMincer is a self-supervised machine learning framework for voltage imaging denoising models. A visual comparison of voltage imaging data before and after CellMincer below:
 
-Installation
-============
+![Alt Text](./docs/source/_static/graphics/raw_denoised_traces.gif)
 
-```
-git clone https://github.com/broadinstitute/CellMincer.git
-pip install -e CellMincer/
-```
+A schmatic of Optopatch data and CellMincer's high-level architecture below:
```

Comment:
An overview of voltage imaging data generation and CellMincer denoising model is shown below:

(image)

(a) A simplified schematic diagram of a typical optical voltage imaging experiment (left). The spatially resolved fluorescence response is recorded over time to produce a voltage imaging movie. A key component of CellMincer's preprocessing pipeline is the computation of spatial summary statistics and various auto-correlations from the entire recording, which are concatenated into a stack of global features (right). (b) An overview of CellMincer's deep learning architecture. (c) The conditional U-Net convolutional neural network (CNN). At each step in the contracting path, the precomputed global feature stack is spatially downsampled in parallel ($\mathscr{F} \rightarrow \mathscr{F}' \rightarrow \mathscr{F}'' \rightarrow \ldots$) and concatenated to the intermediate spatial feature maps. (d) The temporal post-processor neural network. The sequence of pixel embeddings are convolved with a 1D kernel along the time dimension, producing a single vector of length $C$. A multilayer perceptron subsequently reduces this vector to a single value. (e) A comparison of model performance on simulated data before and after introducing global features as a U-Net conditioner. The distributions of PSNR gain are binned by stimulation amplitude. Using global features confers an average increase of 5 dB to the denoiser, roughly corresponding to a 3-fold noise reduction.